libvmaf/meson_options.txt

```diff
@@ -27,3 +27,13 @@ option('enable_float',
     type: 'boolean',
     value: false,
     description: 'Compile floating-point feature extractors into the library')
+
+option('enable_float_funque',
+    type: 'boolean',
+    value: false,
+    description: 'Compile floating-point funque feature extractors into the library')
+
+option('funque_double_dtype',
```

Comment:
I think we can simplify this and make double the default, as discussed the float32 usage in the Python version was incidental rather than intentional. We can update the Python code to also use double for parity.

Reply:
Since Float and Double have scores matching upto 5th decimal places float data type has been finalized

Float datatype is used for all buffer (Spatial filter, DWT bands), intermediate squares and accumulators have been double  (in the float version), these buffers have to be double to maintain precession, in Integer implementation double datatype buffers will be converted to int64